packages/x-charts-vendor/README.md

```diff
@@ -13,27 +13,25 @@ To help provide an easy path to folks still using CommonJS in their Node.js appl
 ## Main difference with victory-vendor
 
 Victory is using the `d3-voronoi` which is an archived project.
-Our chart library relies on the `d3-delaunay` which is also ESM only and reuse `robust-predicates` which is also ESM only
+Our chart library reuses `robust-predicates` which is ESM only.
```

Comment:
The `robust-predicates` was added because it's required by the `d3-delaunay`.

Since your remove the `d3-delaunay` you should also be able to remove the `robust-predicates` from `@mui/x-charts-vendors`.

Might woth checking that the new dependency does not require to be added in the vendor.

If not, you can complitely remove this section